pipelineutilities/pipelineutilities/search_files.py

```diff
@@ -349,6 +351,24 @@ def json_serial(obj):
     raise TypeError("Type %s not serializable" % type(obj))
 
 
+def _add_more_file_fields(json_record: dict, iiif_image_service_uri: str = None) -> dict:
+    """ Add mimeType (if absent), add mediaServer and mediaResourceId """
+    file_path = json_record.get('filePath')
+    if file_path:
+        file_extension = Path(file_path).suffix
+        if file_extension and '.' in file_extension and file_extension.lower() in ['.tif']:
```

Comment:
Is the '.' comparison necessary? Could this be shortened to:
if file_extension and file_extension.lower() in ['.tif']: